### PR TITLE
Compatibility with PHP 8.1 & 8.2

### PR DESCRIPTION
**Pull request by javierabion**
Changes needed to allow PHP 8.1 compatibillity.
Drop support for PHP versions under 7.2

### CODE REVIEW

**Comment by javierabion** [comment]
Dragos: when you merge, please update version nuembre from 1.1.0 to 1.1.1. Thanks!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,12 +12,11 @@ jobs:
     strategy:
       matrix:
         php-versions:
-          - 5.6
-          - 7.1
           - 7.2
           - 7.3
           - 7.4
           - 8.0
+          - 8.1
```
Comment (dragosprotung) [inline .github/workflows/build.yml:19]: Please add 8.2 as well

File: tests/Unit/ReflectionClass/ReflectionClassTest.php [hunk 2]
```diff
@@ -46,7 +46,7 @@ public function testGetReflectionDocCommentWithCommentReturnDocComment()
     {
         $reflection = new ReflectionClass('\Wingu\OctopusCore\Reflection\Tests\Unit\Fixtures\TestClass3');
         $actual = $reflection->getReflectionDocComment();
-        $expected = new ReflectionDocComment("/**\n *\n * The short definition of test class 3.\n * This the long description of test class 3\n * @author test\n *\n */");
+        $expected = new ReflectionDocComment("/**\r\n *\r\n * The short definition of test class 3.\r\n * This the long description of test class 3\r\n * @author test\r\n *\r\n */");
```
Comment (dragosprotung) [inline tests/Unit/ReflectionClass/ReflectionClassTest.php:49]: Tests are failing becuase of added `\r`

**Review by dragosprotung — CHANGES_REQUESTED** [review]

File: tests/Unit/ReflectionClass/ReflectionClassTest.php (hunk 2 above)
Comment (javierabion) [reply tests/Unit/ReflectionClass/ReflectionClassTest.php]: Resolved in new commit

**Review by javierabion — COMMENTED** [review]
Everything solved. Thanks!

**Review by dragosprotung — APPROVED** [review]

**Comment by dragosprotung** [comment]
Thank you @javierabion

**Comment by javierabion** [comment]
You're wellcome! @dragosprotung

**Pull request merged**